MCMC/MCMC-flow/project.py

```diff
@@ -131,6 +134,33 @@ def sample(job):
         print("Simulation finished completed")
         print("-----------------------------")
 
+@MyProject.operation
+@MyProject.pre(sampled)
+@MyProject.post(analyzed)
+def analysis(job):
+    from utils import avg_nn, rdf
+    import numpy as np
+    os.makedirs(os.path.join(job.ws, "analysis/rdf"))
+    gsdfile = job.fn('trajectory_1.gsd')
+    rdf = rdf(gsdfile, start=-30)
+    x = rdf.bin_centers
+    y = rdf.rdf
+    peakx = max(x)
+    peaky = max(y)
```

Comment:
I think first and second peaks are more information rich in rdf. The max of bin_centers is always going to be the last value in the list (largest distance) and max of rdf doesn't give us a correct estimation of the first peak.
Previously I used `scipy.signal.find_peaks` from scipy package to find the peak. 
@chrisjonesBSU Feel free to correct me if I'm wrong.

Reply:
I agree. We should use kind of peak finding algorithm or tool. RDFs often have more than 1 peak, so just finding the max y value may not provide enough information.